doc/src/main.ltx

```diff
@@ -54,20 +54,11 @@
 
 \begin{document}
 
-% Center the title page
-\null
-\nointerlineskip
-\vfill
-\let\snewpage \newpage
-\let\newpage \relax
-\maketitle
-\textit{This report represents the work of WPI undergraduate students submitted
-to the faculty as evidence of completion of a degree requirement. WPI routinely
-publishes these reports on its website without editorial or peer review. For
-more information about the projects program at WPI, please see
-\url{http://www.wpi.edu/academics/ugradstudies/project-learning.html}}
-\let \newpage \snewpage
-\vfill 
+\input{title.ltx}
+
+\newpage
+\thispagestyle{empty}
+\mbox{}
 \newpage
 
```

Comment:
Add right here (restarts page numbering for the abstract, so it's 1, instead of 2):

% Restart page numbering
\setcounter{page}{1}